engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/WorkflowExecutionActor.scala

```diff
@@ -251,7 +251,7 @@ case class WorkflowExecutionActor(params: WorkflowExecutionActorParams)
     // - The job lasted too long (eg PAPI 6 day timeout)
     // Treat it like any other non-retryable failure:
     case Event(JobAbortedResponse(jobKey), stateData) =>
-      val cause = new Exception("The job was aborted from outside Cromwell")
+      val cause = new Exception(s"The compute backend terminated the job. Likely reasons are preemption, running out of disk or memory on the compute instance, or exceeding the backend's maximum job duration.")
```

Comment:
In HPC environments (or PAPI I suppose if they know the API calls to make), it can also indicate the user stopping their own job manually.

Not sure that's worth calling out as a separate case though, since in those situations they'd presumably know about it...